src/libm-tester/CMakeLists.txt

```diff
@@ -52,13 +52,38 @@ macro(test_extension SIMD)
       endif (COMPILER_SUPPORTS_WEAK_ALIASES)
     endif(ENABLE_GNUABI)
 
+    # Build tester2 SIMD
+    string(TOLOWER ${SIMD} SCSIMD)
```

Comment:
Do we really need to make it lowercase? My preference would be to avoid this change, so that the naming is uniform all through the project.

Reply:
This is just a matter of preference, but if I remove TOLOWER here, the file names of the executables would be like tester2AVX2dp. I prefer lower case file names, since it is less confusing. There are case-sensitive and case-insensitive filesystems.

Reply:
I see, it makes as sense. Thanks